### ISSUE TITLE
BUG: set_tick_params labelcolor should apply to offset

### ISSUE THREAD

**Issue by astrofrog** [matplotlib/matplotlib#6440]
The following example:

```python
import matplotlib.pyplot as plt

fig = plt.figure()
ax = fig.add_subplot(1,1,1)
ax.plot([1.01e9,1.02e9,1.03e9])
ax.yaxis.set_tick_params(labelcolor='red')
fig.savefig('test.png')
```

produces the following output:

![test](https://cloud.githubusercontent.com/assets/314716/15325061/08eb3230-1c41-11e6-97ea-14da7857a096.png)

The offset label should likely be set by the ``labelcolor`` option too.

**Comment by jenshnielsen** [comment]
Fixed on master by #6445 but this has not been backported

**Comment by QuLogic** [comment]
Fix is now backported.
